src/ChurchCRM/data/Country.php

```diff
@@ -5,28 +5,35 @@ class Country implements \JsonSerializable
 {
   private string $countryCode;
   private string $countryName;
-  private ?string $countryNameYasumi = null;
-  public function __construct (string $CountryCode, string $CountryName, string $CountryNameYasumi = null)
+  private ?string $countryNameYasumi;
+
+  public function __construct(string $CountryCode, string $CountryName, string $CountryNameYasumi = null)
```

Comment:
I realized I missed this, `$CountryNameYasumi` should probably be updated to show that it is nullable:

```suggestion
  public function __construct(string $CountryCode, string $CountryName, ?string $CountryNameYasumi = null)
```

Reply:
I can't find anything that points to the use of ?

Reply:
@DawoudIO Here's a link to the PHP documentation around nullable types: https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.nullable